core/src/main/java/jenkins/util/xml/XMLUtils.java

```diff
@@ -137,16 +139,9 @@ public static void safeTransform(@Nonnull Source source, @Nonnull Result out) th
             throw new IllegalArgumentException(String.format("File %s does not exist or is not a 'normal' file.", file.getAbsolutePath()));
         }
 
-        FileInputStream fileInputStream = new FileInputStream(file);
```

Comment:
IntelliJ highlights (low-lights?) unused imports and makes them trivial to remove.

Still, just wondered, not required IMO.